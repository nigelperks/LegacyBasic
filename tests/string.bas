100 PRINT ">"A$"<"
110 A$="HELLO THERE"
120 PRINT ">"A$"<"
130 PRINT A$+"! "+A$+"!"
140 PRINT ">"LEFT$(A$,0)"<"
150 PRINT ">"LEFT$(A$,5)"<"
160 PRINT ">"LEFT$(A$,50)"<"
170 PRINT ">"RIGHT$(A$,0)"<"
180 PRINT ">"RIGHT$(A$,5)"<"
190 PRINT ">"RIGHT$(A$,50)"<"
200 PRINT ">"MID$(A$,4,6)"<"
210 PRINT ">"MID$(A$,4,0)"<"
220 PRINT ">"MID$(A$,4,50)"<"
230 PRINT ">"MID$(A$,11,1)"<"
240 PRINT LEN(A$)
250 PRINT STR$(3.1416),LEN(STR$(3.1416))
260 N$="2.719e3":PRINT VAL(N$)
270 PRINT CHR$(42)CHR$(42)CHR$(42)ASC("A")ASC("a")
