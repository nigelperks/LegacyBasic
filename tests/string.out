><
>HELLO THERE<
HELLO THERE! HELLO THERE!
><
>HELLO<
>HELLO THERE<
><
>THERE<
>HELLO THERE<
>LO THE<
><
>LO THERE<
>E<
 11 
3.1416   6 
 2719 
*** 65  97 
